bugbug/models/regressionrange.py

```diff
@@ -80,6 +80,11 @@ def get_labels(self):
                     classes[bug_id] = 1
                 elif bug_data["cf_has_regression_range"] == "no":
                     classes[bug_id] = 0
+            # Check the regressed_by field
+            elif "regressed_by" in bug_data and bug_data["regressed_by"]:
```

Comment:
The `cf_has_regression_range` has been removed. We can rely solely on `regressed_by` instead.